src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/ResettlementAssessmentService.kt

```diff
@@ -198,4 +211,26 @@ class ResettlementAssessmentService(
       .map { it.trim() }
       .map { element -> options?.find { it.id == element }?.displayText ?: element }
       .reduceOrNull { acc, value -> "$acc\n$value" } ?: ""
+
+  @Transactional
+  fun skipAssessment(nomsId: String, assessmentType: ResettlementAssessmentType, request: AssessmentSkipRequest) {
+    if (assessmentType != ResettlementAssessmentType.BCST2) {
+      throw ServerWebInputException("Only BCST2 assessment can currently be skipped")
+    }
+
+    val prisonerEntity = getPrisonerEntityOrThrow(nomsId)
+    val assessmentSummary = getAssessmentSummary(prisonerEntity, assessmentType)
+    if (assessmentSummary.any { it.assessmentStatus != ResettlementAssessmentStatus.NOT_STARTED }) {
+      throw ServerWebInputException("Cannot skip assessment that has already been started")
```

Comment:
Can you Skip an assessment by calling this endpoint more than once? I don't see a check for "is already skipped" ?